changelog.d/6849.bugfix

```diff
@@ -0,0 +1 @@
+ Synapse is now tolerant of the federation_certificate_verification_whitelist option being None or not specified.
```

Comment:
```suggestion
Fix Synapse refusing to start if `federation_certificate_verification_whitelist` option is blank.
```

(We tend to try to write bugfix newsfiles starting with "Fix..." and then description of bug)